tools/azsdk-cli/Azure.Sdk.Tools.Cli/README.md

```diff
@@ -1,37 +1,61 @@
 # `Azure SDK CLI`
 
-This implementation is the Azure SDK Engineering System's `everything` server.
-
-It is eventually intended to encapsulate a lot of the manual work in an `azure sdk` package's release process.
-
-This server is intended to be run in one of two ways:
-
-- As a standalone `MCP` server that can be added directly to a `.vscode/mcp.json` file through `stdio`.
-  - Downloaded standalone exe:
-    - ```jsonc
-      // within mcp.json `servers` member:
-      "Azure SDK Everything": {
-        "type": "stdio",
-        "command": "path/to/Azure.Sdk.Tools.Cli.exe",
-        "args": [
-          "start"
-        ]
-      }
-      ```
-  - Directly as a `dotnet tool install`-ed dotnet tool
-    - ```jsonc
-      "Azure SDK Everything": {
-        "type": "stdio",
-        "command": "azsdk-cli",
-        "args": [
-          "start"
-        ]
-      }
-      ```
-- As a standalone tool encapsulating specific tool usage
+The Azure SDK Engineering System's automation server that is intended to encapsulate manual work in the `azure sdk` package's release process.
+
+## Prerequisites
+- .NET 8.0
+- Visual Studio Code 
+    - [Copilot Extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot)
+    - [C# Dev Kit Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit) (optional)
+
+## Quick Start
+
+1. **Open VS Code** in the `azure-sdk-tools/tools/azsdk-cli` directory
+2. **Start the MCP server** (optional - Copilot will auto-start if needed):
+   - In `mcp.json`, click the Start button below "servers"
+   
+   ![Local Image](/tools/azsdk-cli/Azure.Sdk.Tools.Cli/image/MCP-Start.png)
+
+3. **Test the connection** by prompting Copilot (`Ctrl + Shift + I`):
+
+   ```text
+   "Use the hello-world MCP tool to echo back 'Testing the tool'"
+   ```
+
+## Usage Modes
+
+### 1. MCP Server Mode
+
+Add to `.vscode/mcp.json` file through `stdio`:
+
+**Using standalone executable:**
+
+```jsonc
+// within mcp.json "servers" section:
+"Azure SDK Everything": {
+  "type": "stdio",
+  "command": "path/to/Azure.Sdk.Tools.Cli.exe",
+  "args": ["start"]
+}
+```
+
+**Using dotnet tool:**
+
+```jsonc
+"Azure SDK Everything": {
+  "type": "stdio", 
+  "command": "azsdk-cli",
+  "args": ["start"]
+}
```

Comment:
As @michiisai mentioned, this is incorrect w/r/t using the `dotnet` tool. If using dotnet the command should be `dotnet` and the args should be `["run", "--", "start"]`